### ISSUE TITLE
Make shopping in multiple shops possible

### ISSUE THREAD

**Issue by tschumilas** [openfoodfoundation/wishlist#501]
## What is the need/problem?
Today a shopper can only shop in one shop at a time.  If they visit a second shop to begin a cart,  their first cart disappears.
However, the OFN groups feature encourages 'networking'  and introduces customers to multiple shops and hubs on the platform,  but then frustrates users by not taking them there to actively shop.

In Canada,  we have many small shops that link together in a 'group'  for marketing purposes.  Each group  member is a separate enterprise selling their own products.    We want to maximize the marketing impact of a group by allowing customers to shop with multiple group members at the same time.

## What type of users does this affect (and how many, if known)?
This effects primarily 'sells own' enterprises who are not supplying a 'hub' (multi vendor shop).  We experience a lot of friction with setting up hubs in Canada because of the time it takes to run a hub (hub fees do not come close to covering these costs).  

In our experience,  users like groups, and we'd like to further innovate on this existing feature.  We think its powerful and an  under-used feature in OFN today.  

## Potential solutions that could solve the problem?
A shopper can visit multiple open OFN stores,  and have a separate cart in each.  These carts would not disappear when the shopper goes to another store.  
Questions -  how long does the cart persist?   until the OC closes?  only as long as the browzer window is open?

## Additional context
Previously work was done on this [issue](https://github.com/openfoodfoundation/openfoodnetwork/pull/5601).  But it was[ reverted](https://github.com/openfoodfoundation/openfoodnetwork/pull/5770) because we realized that OFN did not support multi-cart 
shopping. 

There have also been discussions about adding a 'split payment' capacity.  In split,  payments,  an order total paid at checkout is 'split' into different supplier wallets.   As I understand it,  Stripe supports this, but not for the way we have enabled Stripe.    Split payments would partially solve the problem -  a 'group' of suppliers could form an OFN hub,  and there would be no need for the hub coordinator to take payments and issue payouts to suppliers.  But,  there would still be the need for the hub to edit orders,  and issue supplementary payments in some way if items are added to orders post checkout,  etc.   And,  split payments would require all suppliers to use an automated payment method (cash payments would not be possible)

So,  for these reasons,  enabling shopping in multiple carts seems a better solution.  Each supplier can setup whatever payment methods and shipping methods they want.

**Comment by RachL** [comment]
@tschumilas note that:

>  Each supplier can setup whatever payment methods and shipping methods they want.

Means that is someone creates a cart with multiple suppliers, they will end up with different payment and shipping solution.

I think we would want to restrict that for the shopper experience?

**Comment by tschumilas** [comment]
Sorry if I wasn't clear -   right now,  when enterprises are grouped with the groups feature,  each enterprise has its own store (sells own or hub),  so each has their own payment/shipping methods (and all other options in their profile....)    I am looking for a way to access and buy from these separate stores without losing each cart.  Today -  I put items from store 1 in my cart.   I visit store 2 and I start a new cart.   I want to be able to keep the cart from store 1 and go back and continue shopping there.   I would checkout from each store separately.

**Comment by RachL** [comment]
@tschumilas only to be sure I understand: so if there are 6 enterprises you want to buy from, you checkout 6 times?

**Comment by tschumilas** [comment]
correct.   The 'group' is just a promotional gateway to the individual farms.
